cylc/flow/network/server.py

```diff
@@ -367,41 +402,24 @@ def graphql(
         Returns:
             object: Execution result, or a list with errors.
         """
-        try:
-            executed: 'ExecutionResult' = schema.execute(
-                request_string,
-                variable_values=variables,
-                context_value={
-                    'resolvers': self.resolvers,
-                    'meta': meta or {},
-                },
-                backend=CylcGraphQLBackend(),
-                middleware=list(instantiate_middleware(self.middleware)),
-                executor=AsyncioExecutor(),
-                validate=True,  # validate schema (dev only? default is True)
-                return_promise=False,
-            )
-        except Exception as exc:
-            return 'ERROR: GraphQL execution error \n%s' % exc
```

Comment:
Where does this error get caught?

Reply:
This function returning a string was broken as it lead to
```console
$ cylc trigger wflow//20250221T0000Z/foo
Error processing command:
    AttributeError: 'str' object has no attribute 'values'
```
with nothing in the scheduler log